src/index.js

```diff
@@ -842,7 +858,15 @@ async function start() {
       case "users": {
         // "!hubs users" == list users
         if (hubState != null) {
-          const names = Object.values(hubState.reticulumCh.getUsers()).map(info => info.metas[0].profile.displayName);
+          const names = Object.values(hubState.reticulumCh.getUsers()).map(
+            info => {
+              let profile = info.metas[0]?.profile;
```

Comment:
If we ought to record when a user record is malformed, feel free to change the optional chaining to an if ... else console.warn(..)

I'm fine either way.

Reply:
Interesting idea, but I don't think it would help all that much because the person using the command probably isn't going to have the logs for the bot open at the same time.  There's probably a way to generate a Discord message with a warning, though.  Hmm.